### PR TITLE
updated 4020 4022

### PR DESCRIPTION
**Pull request by eu-genia**
https://github.com/BetaMasaheft/Documentation/issues/1578

### CODE REVIEW

File: 4001-5000/LIT4020RegulaeHierarchia.xml [hunk 1]
```diff
@@ -5,6 +5,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <fileDesc>
          <titleStmt>
             <title xml:id="t1">Regulae de Hierarchia ecclesiastica</title>
+            <title xml:id="t2" xml:lang="gez">ሥርዓተ፡ ቤተ፡ ክርስቲያን፡ ወማዕርጊሃ፡</title>
```
Comment (MarcinKrawczuk) [inline 4001-5000/LIT4020RegulaeHierarchia.xml:8]: The plural of ማዕርግ māʿǝrǝg is መዓርግ maʿārǝg, shouldn't we standardize it?

File: 4001-5000/LIT4022RegulaeFidei.xml [hunk 2]
```diff
@@ -5,6 +5,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <fileDesc>
          <titleStmt>
             <title xml:id="t1" xml:lang="la" type="supplied">Regulae fidei et vitae</title>
+            <title xml:id="t2" xml:lang="gez">አንቀጽ፡ ቀዳማዊ፡ በእንተ፡ ቤተ፡ ክርስቲያን፡ ወዘኵሉ፡ ሥርዓታ።</title>
+            <title corresp="#t2" xml:lang="gez" type="normalized">ʾAnqaṣ qǝdāmāwi ba-ʾǝnta beta krǝstiyān wa-za-kwǝllu śǝrʿātā</title>
```
Comment (MarcinKrawczuk) [inline 4001-5000/LIT4022RegulaeFidei.xml:9]: qadāmāwi ... kʷǝllu

**Review by MarcinKrawczuk — CHANGES_REQUESTED** [review]
I've never done a review before, so I have just to transliteration comments for you consideration.

File: 4001-5000/LIT4020RegulaeHierarchia.xml (hunk 1 above)
Comment (eu-genia) [reply 4001-5000/LIT4020RegulaeHierarchia.xml]: thank you, donw

**Comment by eu-genia** [comment]
@MarcinKrawczuk if all is OK now just click review and approve, thank you

**Review by MarcinKrawczuk — APPROVED** [review]

**Pull request merged**